service/src/main/java/uk/gov/hmcts/reform/fpl/events/CallbackEvent.java

```diff
@@ -1,29 +1,27 @@
 package uk.gov.hmcts.reform.fpl.events;
 
 import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.fpl.request.RequestData;
 
 public class CallbackEvent {
 
-    //TODO TECHDEBT use RequestData here so that we don't pass authorization/user id into every event FPLA-1475
     private final CallbackRequest callbackRequest;
-    private final String authorization;
-    private final String userId;
+    private final RequestData requestData;
 
-    CallbackEvent(CallbackRequest callbackRequest, String authorization, String userId) {
+    CallbackEvent(CallbackRequest callbackRequest, RequestData requestData) {
         this.callbackRequest = callbackRequest;
-        this.authorization = authorization;
-        this.userId = userId;
+        this.requestData = requestData;
```

Comment:
Not sure you need to store request data, you could just extract the userId and authorization here instead

Reply:
Fixed